src/midiEditorCommands.cpp

```diff
@@ -605,15 +627,12 @@ pair<int, int> findChord(MediaItem_Take* take, int direction) {
 // It is not a REAPER note index!
 // -1 means not in a chord.
 int curNoteInChord = -1;
-// Keeps track of the CC to which the user last moved.
-// This is a REAPER CC index.
-// -1 means no CC.
-int currentCC = -1;
-
-// Used to order notes in a chord by pitch.
-bool compareNotesByPitch(const MidiNote& note1, const MidiNote& note2) {
-	return note1.pitch < note2.pitch;
-}
+// Keeps track of the CC to which the user last moved at a particular position.
+// The first entry of the pair is the position.
+// The second entry of the pair is the number of the CC at the position; e.g. 0 is the first CC.
+// It is not a REAPER CC index.
+// -1 means no position/CC.
+pair<double, int> currentCCAtPos = {-1, -1};
```

Comment:
I'm not quite sure about the name currentCCAtPos. On one hand, I understand this now includes a position. On the other hand, it feels like currentCC is still a better name. It's not the current CC at a particular position; it's the current CC for the user, including both its position and index at that position. I think I'd prefer either currentCC (which would require renaming some local variables below) or currentCCAndPos.